the-9-exercises/src/App.jsx

```diff
@@ -1,14 +1,23 @@
-
-import './App.css'
+import List from "./components/List";
 
 function App() {
-
+  const arrayOfItems = [
+    "dog  ",
```

Comment:
You don't have to mention the type of the variable in its variable name. It makes the name unnecessarily long and you can tell its type by just looking at the code. Like `arrayOfItems` can be simplified to `items` by putting it in plural already means it's an array (a group of things)